src/PythonClient/src/quixstreams/models/streamproducer/streampropertiesproducer.py

```diff
@@ -13,16 +13,16 @@
 @nativedecorator
 class StreamPropertiesProducer(object):
     """
-        Provides additional context for the stream
+    Represents properties and metadata of the stream.
+    All changes to these properties are automatically populated to this class.
```

Comment:
This would be true for consumer.

StreamPropertiesProducer 
Represents properties and metadata of the stream.
All changes to these properties are automatically published to the underlying stream.